src/test/regress/expected/single_node_truncate.out

```diff
@@ -0,0 +1,131 @@
+CREATE SCHEMA single_node_truncate;
+SET search_path TO single_node_truncate;
+SET citus.shard_replication_factor TO 1;
+-- helper view that prints out local table names and sizes in the schema
+CREATE VIEW table_sizes AS
+SELECT
+  c.relname as name,
+  pg_catalog.pg_table_size(c.oid) > 0 as has_data
+FROM pg_catalog.pg_class c
+     LEFT JOIN pg_catalog.pg_namespace n ON n.oid = c.relnamespace
+WHERE c.relkind = 'r'
+      AND n.nspname = 'single_node_truncate'
+ORDER BY 1;
+-- test truncating reference tables
+CREATE TABLE ref(id int UNIQUE, data int);
+INSERT INTO ref SELECT x,x FROM generate_series(1,10000) x;
+SELECT create_reference_table('ref');
+NOTICE:  Copying data from local table...
+NOTICE:  copying the data has completed
+DETAIL:  The local data in the table is no longer visible, but is still on disk.
+HINT:  To remove the local data, run: SELECT truncate_local_data_after_distributing_table($$single_node_truncate.ref$$)
+ create_reference_table
+---------------------------------------------------------------------
+
+(1 row)
+
+CREATE TABLE citus_local(id int, ref_id int REFERENCES ref(id));
+INSERT INTO citus_local SELECT x,x FROM generate_series(1,10000) x;
+-- verify that shell tables for citus local tables are empty
```

Comment:
It may be worth throwing in a truncate_local_data_after_distributing_table call on a Citus local table just to make sure it does not do something horrible (although it should never be necessary).

Reply:
Done.